content/concept-escrow.md

```diff
@@ -0,0 +1,89 @@
+# Escrow
+
+Escrow is a feature of the XRP Ledger that allows you to send conditional XRP payments. These conditional payments, called _escrows_, set aside XRP and deliver it later when certain conditions are met. Conditions to successfully finish an escrow include time-based unlocks and [crypto-conditions][]. Escrows can also be set to expire if not finished in time. Conditional held payments are a key feature for full [Interledger Protocol][] support, which enables chains of payments to cross any number of ledgers.
+
+The XRP set aside in an escrow is locked up. No one can use or destroy the XRP until the escrow has been successfully finished or canceled. Before the expiration time, only the intended receiver can get the XRP. After the expiration time, the XRP can only be returned to the sender.
+
+## Usage
+
+<!--{# Diagram sources: https://docs.google.com/presentation/d/1C-_TLkkoQEH7KJ6Gjwa1gO6EX17SLiJ8lxvFcAl6Rxo/ #}-->
+
+[![Escrow Flow Diagram (Successful finish)](img/escrow-success-flow.png)](img/escrow-success-flow.png)
+
+**Step 1:** To send an escrow, the sender uses an [EscrowCreate transaction][] to set aside some XRP. This transaction defines a finish time, an expiration time, or both. The transaction may also define a crypto-condition that must be fulfilled to finish the escrow. This transaction must define an intended recipient for the XRP; the recipient _may_ be the same as the sender.
+
+**Step 2:** After this transaction has been processed, the XRP Ledger has an [Escrow object](reference-ledger-format.html#escrow) that holds the escrowed XRP. This object tracks the properties of the escrow as defined by the transaction that created it. If this escrow has a finish time, no one can access the XRP before then.
+
+**Step 3:** The recipient, or any other XRP Ledger address, sends an [EscrowFinish transaction][] to deliver the XRP. If the correct conditions are met, this destroys the Escrow object in the ledger and credits the XRP to the intended recipient. If the escrow has a crypto-condition, this transaction must include a fulfillment for that condition. If the escrow has an expiration time, this cannot happen after that time.
```

Comment:
Wow, so, anyone could finish an escrow transaction, even if they're not the sender or recipient? That's wild.

In the last sentence, 'this cannot happen after that time', I'd suggest just stating again to what 'this' refers:

>  If the escrow has an expiration time, condition fulfillment cannot happen after that time.

Reply:
> Wow, so, anyone could finish an escrow transaction, even if they're not the sender or recipient? That's wild.

Yep, it's a little surprising. On the other hand, there's not really any downside other than it being unexpected: if someone else finishes an escrow for you, they're just saving you the effort and the XRP equivalent of a postage stamp. In the case of a conditional escrow, it's unlikely that anyone other than the sender or recipient can actually fulfill the condition, anyway.